rero_ils/modules/operation_logs/jsonschemas/operation_logs/operation_log-v0.0.1.json

```diff
@@ -78,6 +78,187 @@
           "pattern": "^https://ils.rero.ch/api/organisations/.+?$"
         }
       }
+    },
+    "loan": {
+      "title": "Loan",
+      "description": "Loan specific data",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "override_flag": {
+          "title": "Override flag",
```

Comment:
It would be great to have a `description` here to have more details on this field.